tests/apichecks/testdata/exceptions/missingrefs.txt

```diff
@@ -46,6 +46,7 @@
 [refs] crd=essentialcontactscontacts.essentialcontacts.cnrm.cloud.google.com version=v1alpha1: field ".spec.parent" should be a reference
 [refs] crd=gkebackupbackupplans.gkebackup.cnrm.cloud.google.com version=v1alpha1: field ".spec.backupConfig.encryptionKey.gcpKmsEncryptionKey" should be a reference
 [refs] crd=iapsettings.iap.cnrm.cloud.google.com version=v1alpha1: field ".spec.name" should be a reference
+[refs] crd=iapsettings.iap.cnrm.cloud.google.com version=v1beta1: field ".spec.name" should be a reference
```

Comment:
+1 to the missing ref catch, I don't think we want to make this exception for beta resource